### PR TITLE
Web feature/google auth

### PR DESCRIPTION
**Pull request by afraarslan**
#120 started. 
* I designed a component for Google SignIn/Signout. It is added on the Login page. 
* We decided to use the **Grouping by file type** file structure for our project. I made some changes according to this structure.

Note:
The backend side of the component is still in progress.

### CODE REVIEW

File: web/carousel/package.json [hunk 1]
```diff
@@ -3,12 +3,20 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.32",
+    "@fortawesome/free-brands-svg-icons": "^5.15.1",
+    "@fortawesome/free-regular-svg-icons": "^5.15.1",
+    "@fortawesome/free-solid-svg-icons": "^5.15.1",
+    "@fortawesome/react-fontawesome": "^0.1.12",
```
Comment (afraarslan) [inline web/carousel/package.json:10]: Install [Fontawesome](https://fontawesome.com/) library for icons. I used `faGoogle` icon in this library.

File: web/carousel/src/App.js [hunk 2]
```diff
@@ -1,28 +1,10 @@
...
-
   return (
-    <div className="App">
-      <header className="App-header">
-        <p>
-          Hi Carousel!!
-        </p>
-        <ProductTable productList={productList} columnSize={3}/>
-
-      </header>
```
Comment (afraarslan) [inline web/carousel/src/App.js:25]: Moved into the Home page

File: web/carousel/src/screens/Login/index.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import { GoogleAuth } from "../../components/GoogleAuth";
+
+export default function Login() {
+  return <GoogleAuth />;
```
Comment (afraarslan) [inline web/carousel/src/screens/Login/index.js:4]: GoogleAuth component added on the Login page.

File: web/carousel/src/screens/index.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+const App = () => {
+  return (
+    <>
+      {/* <Header /> */}
+      <Router>
+        <Switch>
+          <Route path="/" exact component={Home} />
+          <Route path="/login" exact component={Login} />
+        </Switch>
+      </Router>
```
Comment (afraarslan) [inline web/carousel/src/screens/index.js:16]: Router for Pages is defined here. We can also add `Header` component after we implement it in components/Header

**Review by oncelkeles — DISMISSED** [review]
Thank you Afra! Great work 🔥

**Review by ahmetyigitgedik — APPROVED** [review]
Thanks a lot for your effort! Liked very much :stuck_out_tongue:

**Review by ahmetyigitgedik — APPROVED** [review]

**Pull request merged**